clouddriver-cloudfoundry/src/test/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/ServiceInstancesTest.java

```diff
@@ -32,23 +32,20 @@
 import java.util.Collections;
 
 import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.ErrorDescription.Code.SERVICE_ALREADY_EXISTS;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.State.FAILED;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.State.IN_PROGRESS;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.State.SUCCEEDED;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.Type.CREATE;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.Type.DELETE;
-import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.Type.UPDATE;
-import static junit.framework.TestCase.fail;
+import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.State.*;
+import static com.netflix.spinnaker.clouddriver.cloudfoundry.client.model.v2.LastOperation.Type.*;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
 import static org.mockito.Mockito.*;
 
-public class ServiceInstancesTest {
+class ServiceInstancesTest {
   private ServiceInstances serviceInstances;
   private CloudFoundrySpace cloudFoundrySpace;
   private ServiceInstanceService serviceInstanceService;
 
-  @Before
-  public void before() {
+  @BeforeEach
+  void before() {
     serviceInstanceService = mock(ServiceInstanceService.class);
```

Comment:
You could just initialize with the mocks above and avoid a  BeforeEach entirely if this is all it is needed for. I tend to use BeforeEach for doing nontrivial preparation of the class under test.

Reply:
True, I'll fix it.

Reply:
Changed.